core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -233,7 +244,12 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
       val nowUuid = UUIDs.timeBased()
       val now = UUIDs.unixTimestamp(nowUuid)
 
-      preparedWriteMessage.map { stmt =>
+      // using two separate statements with or without the meta data columns because
+      // then users doesn't have to alter table and add the new columns if they don't use
+      // the meta data feature
```

Comment:
perhaps explicitly say "users who created their tables with version v0.54 or earlier don't have to alter table"?